style.css

```diff
@@ -442,3 +462,393 @@ ul > li > i {
   margin-top: 22px;
   text-align: center;
 }
+
+@media only screen and (min-width: 992px) {
+  body {
+    font-family: 'Poppins', sans-serif;
+    margin: 0;
+    background-color: #e5e5e5;
+  }
+
+  header {
+    width: 100%;
+    height: 916px;
+    background-color: #fff;
+  }
+
+  .top-navigation {
+    width: 1440px;
+    height: 72px;
+  }
+
+  .logo {
+    width: 100px;
+    height: 40px;
+    margin: 25px 0 0 155px;
+  }
+
+  .item-navigation {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    text-decoration: none;
+    width: 259px;
+    height: 40px;
+    margin-right: 142px;
+  }
+
+  [alt*="hamburger"] {
+    display: none;
+  }
+
+  .headline {
+    width: 1440px;
+    height: 721.46px;
+    background-image: url(images/Header-bg.svg);
+    background-repeat: no-repeat;
+  }
+
+  li {
+    list-style-type: none;
+    text-decoration: none;
+  }
+
+  .headline h1 {
+    width: 920px;
+    height: 122px;
+    margin: 200px 260px 0 260px;
+  }
+
+  .presentation {
+    width: 920px;
+    height: 98px;
+    font-size: 20px;
+    line-height: 28px;
+    padding: 0 260px;
+    margin: 12px 0 0 0;
+  }
+
+  .connect {
+    height: 24px;
+    padding: 0 260px;
+    margin: 12px 0;
+  }
+
+  .social-media {
+    width: 170.17px;
+    height: 20px;
+    padding: 0 260px;
+    margin: 12px 0;
+  }
+
+  .works {
+    width: 1156px;
+    align-items: center;
+    padding: 142px;
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    background-color: #e5e5e5;
+    grid-gap: 88px;
+  }
+
+  .work-card {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    border: 1px solid #dfe1e6;
+    margin: auto;
+    padding: 16px;
+    border-radius: 24px;
+    width: 1156px;
+    height: 496px;
+    background: #fff;
+  }
+
+  .reverse-card {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .profile-image {
+    margin: 12px 0;
+    width: 544px;
+    height: 448px;
+  }
+
+  .left-block {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 24px 22px;
+  }
+
+  .project-title {
+    width: 496px;
+    height: 46px;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    margin: 20px;
+  }
+
+  .language-list {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 297px;
+    height: 40px;
+  }
+
+  .credentials-list {
+    height: 40px;
+    align-items: center;
+  }
+
+  .language-list li {
+    align-items: center;
+    background-color: #ebebff;
+    border-radius: 8px;
+    font-weight: 500;
+  }
```

Comment:
- [ ] Let's fix this as there seems to be wrong padding here. Consider checking Figma to see the padding and other properties for this element and add them so that it displays properly on both desktop and mobile version.
- [ ] Let's correct a typo error also. It's `Ruby on rails` and not `Ruby and rails` 💪 

Your page:
<img width="166" alt="Loom_oc9pInxhYB" src="https://user-images.githubusercontent.com/81468675/149593440-7213374d-19e7-423d-ac10-60ec88cea7d7.png">

Figma:
<img width="118" alt="Loom_DDqaLksR2q" src="https://user-images.githubusercontent.com/81468675/149593830-3d9219f9-0404-4940-b1f8-a6690103e57f.png">

Reply:
Resolved